priv/repo/migrations/20250420174258_create_jac_delhi_cutoff_2023.exs

```diff
@@ -0,0 +1,18 @@
+defmodule Dbservice.Repo.Migrations.CreateJacDelhiCutoff2023 do
+  use Ecto.Migration
+
+  def change do
+    create table(:jac_delhi_cutoff_2023) do
```

Comment:
Are we planning to create a new table and schema for each year's data? Feels like there should be a better solution.